tests/unit/test_callable_generator.py

```diff
@@ -71,7 +71,7 @@ def test_mapper_with_pruning(self):
         t = CallableGeneratorTest.Temp()
 
         # Head node
-        node = Node(None, None)
+        node = TransformationProxy(Node(None, None))
```

Comment:
Same as before, should we pass `node.proxied_node` to `CallableGenerator`?

Reply:
Yes, see my previous comment